snippets/ponylang-mode/if.block

```diff
@@ -4,5 +4,5 @@
 # contributor: Sean T. Allen <sean@monkeysnatchbanana.com>
 # --
 if ${1:condition} then
-  ${2:if_block}
-end$0
\ No newline at end of file
+  $0
```

Comment:
this looks like an error to me, but i don't remember ya-snippet syntax. what does this change do?